Application/src/view/visuals/Draw.java

```diff
@@ -138,7 +121,8 @@ public void addClassDiagramClass (String name){
      * @param device
      */
     public void addDeploymentDiagramClass (String device){
-        allDeploymentClasses.add(new DeploymentDiagramClass(device));
+        if (!allClassDiagramClasses.contains(device))
```

Comment:
wat

Reply:
you sure this is correct?

Reply:
@IsabelleTornqvist yes. Due to the information I get from the backend
otherwise there will be duplicate ones.

Reply:
oki